app/containers/Organisations/UserOrganisation.tsx

```diff
@@ -11,10 +11,10 @@ export const UserOrganisationComponent = props => {
   }
 
   const statusBadgeColor = {
-    attention: 'warning',
+    rejected: 'warning',
     pending: 'info',
-    expired: 'danger',
-    active: 'success'
+    // Expired: 'danger',
+    approved: 'success'
   };
 
   /** TODO: INTENDED FOR DEMO ONLY. To be removed */
```

Comment:
To properly the new approval interface, while keeping this behavior for development, I suggest that we create two buttons: one that sets directly sets the status as active, only visible in development (`process.env.NODE_ENV !== 'production'`) and one that creates a pending application.

Reply:
good idea